livekit-plugins/livekit-plugins-elevenlabs/livekit/plugins/elevenlabs/tts.py

```diff
@@ -101,43 +101,34 @@ async def list_voices(self) -> List[Voice]:
             data = await resp.json()
             return dict_to_voices_list(data)
 
-    def synthesize(
+    async def synthesize(
```

Comment:
This remains unchanged inside the base class
function signatures are not consistent